build_tools/github_actions/runner/gcp/create_image.sh

```diff
@@ -145,11 +145,13 @@ function create_image() {
         local machine_type="${CPU_MACHINE_TYPE}"
         local image_size_gb="${CPU_IMAGE_SIZE_GB}"
         local maintenance_policy=MIGRATE
+        local -a extra_args=()
         ;;
       gpu)
         local machine_type="${GPU_MACHINE_TYPE}"
         local image_size_gb="${GPU_IMAGE_SIZE_GB}"
         local maintenance_policy=TERMINATE
+        local -a extra_args=("--accelerator=count=1,type=nvidia-tesla-t4")
```

Comment:
How will we create A100 image after this change?

Reply:
It's the same image. It works for both